app/app.go

```diff
@@ -536,22 +537,27 @@ func NewWasmApp(
 		slashingtypes.ModuleName,
 		evidencetypes.ModuleName,
 		stakingtypes.ModuleName,
-		ibchost.ModuleName,
-	)
-	app.mm.SetOrderEndBlockers(
```

Comment:
A lot new modules were added here.
Were they missing before or does it now just require everyone to have an empty end blocker implementation?

Reply:
All modules need to be listed now